lib/dao.js

```diff
@@ -2824,6 +2824,14 @@ function(data, options, cb) {
 
   var allowExtendedOperators = connector.settings &&
     connector.settings.allowExtendedOperators;
+  if (options.hasOwnProperty('allowExtendedOperators')) {
+    allowExtendedOperators = options.allowExtendedOperators === true;
```

Comment:
Could you please add few comments to this code to explain the different behaviour of "options" vs. "model config" in respect to "connector" settings? I am worried that future readers may be puzzled why each branch uses a slightly different approach.